doc/sphinx/proofs/writing-proofs/proof-mode.rst

```diff
@@ -487,11 +487,20 @@ Name a set of section hypotheses for ``Proof using``
 Proof modes
 -----------
 
-When entering proof mode through commands such as :cmd:`Goal` and :cmd:`Proof`,
-Rocq picks by default the |Ltac| mode. Nonetheless, there exist other proof modes
-shipped in the standard Rocq installation, and furthermore some plugins define
-their own proof modes. The default proof mode used when opening a proof can
-be changed using the following option.
+When entering proof mode through commands such as :cmd:`Goal` and
+:cmd:`Proof`, Rocq picks by default the |Ltac| mode. Nonetheless,
+there exist other proof modes shipped in the standard Rocq
+installation, and furthermore some plugins define their own proof
+modes. The default proof mode used when opening a proof can be changed
+using the following option.
```

Comment:
I would have moved the last sentence down to its own paragraph, so it stays right before the option

Reply:
you can make a PR if you want